packages/dev/docs/pages/index.mdx

```diff
@@ -10,8 +10,37 @@ governing permissions and limitations under the License. -->
 import {Layout} from '@react-spectrum/docs';
 export default Layout;
 
+import {Highlights, Image} from '@react-spectrum/docs';
+import heroNarrow from 'url:./assets/hero-narrow.png';
+import heroWide from 'url:./assets/hero-wide.png';
+
 # React Spectrum
 
-- [React Spectrum](./react-spectrum/index.html)
-- [React Aria](./react-aria/index.html)
-- [React Stately](./react-stately/index.html)
+A collection of libraries and tools that help you build adaptive, accessible, and robust user experiences.
+
+<Image type="hero">
+  <img srcset={`${heroNarrow} 600w,
```

Comment:
hrm... these are coming up as broken links

Reply:
@devongovett is looking into it